### ISSUE TITLE
More descriptive tab titles

### ISSUE THREAD

**Issue by M-Miller-42** [mapbox/osmcha-frontend#578]
## I'm submitting a feature request

At the moment the tab title is always `OSMCha`. It would be useful to be more specific.

## What is the motivation / use case for this feature?
Many browser features are hard to use at the moment, because they rely on descriptive tab titles:
 - If you have multiple OSMCha tabs open, they all look the same in the tab bar
 - The browser history and holding the back button shows no useful information 

## What  is the current behaviour, (attach relevant screenshots) ?
![Screenshot from 2021-08-15 17-55-40](https://user-images.githubusercontent.com/4184356/129484884-5574bf35-afcf-4f86-b39c-9ba458bc13ab.png)

## What  is the expected behaviour ?
I suggest showing the name of the current filter for named filters. 
For unnamed filters (for example when clicking a username) the most important filter parameters could be shown, e.g. `OSMCha  - Flagged from <username>`

## Please tell us about your environment:
Firefox 91.0 (64-bit)

**Comment by Lee-Carre** [comment]
Absolutely! The navigation usability (at present, as M-Miller-42 aptly illustrates) is horrendous.

Effective (let alone _efficient_) navigation requires knowing where you've already been (& come from).
The back button is a safety feature for users; what should be an easy way to escape to somewhere more familiar.

Aside from navigation history, another (comparable) scenario which encounters the same problem is if one had multiple tabs, each pointing to (different URLs at) OSMCha.
Which tab is the one that the user wants?

Some relevant usability advice: [Microcontent: how to write headlines, page titles, and subject lines](//www.nngroup.com/articles/microcontent-how-to-write-headlines-page-titles-and-subject-lines/).

Though, a couple minor quibbles:

> tab title

It's actually (the contents of) the HTML `<title>` element (in the `<head>` element).
How it's rendered by an HTTP User-Agent ( e.g. browser) is a different matter. Not all browsers have tabs (and not all HTTP User-Agents have a GUI, or even an interactive UI).

Might I suggest renaming this ticket to **page** titles (that is; `s/tab titles/page titles/`)?

> OSMCha **-** Flagged from <username>

The correct typographical character, in that context, would be the Em-Dash (U+2014). The hyphen character is for compounding words.

So, instead, the quoted sample would be

    OSMCha — Flagged from <username>

---

A secondary benefit of resolving the OP problem, is that server-load might (depending on other factors) reduce.

How? I find that I have to step back one URL at a time, because it's unclear which is the right one to select in the list. That's many more requests to the server than if I can jump back to exactly where I want to return (skipping all the intermediaries).
